composer.json

```diff
@@ -22,11 +22,11 @@
         "php": "^7.2",
         "doctrine/dbal": "^2.6",
         "erusev/parsedown-extra": "~0.7",
-        "kriswallsmith/assetic": "~1.3",
+        "assetic/framework": "dev-2.0/dev",
         "patchwork/jsqueeze": "~2.0",
         "oyejorge/less.php": "~1.7",
         "leafo/scssphp": "~0.6",
-        "symfony/yaml": "^4.0",
+        "symfony/yaml": "^3.4",
```

Comment:
Why was this change required?

Reply:
@LukeTowers Symfony/Yaml 4.0 and above do not allow numeric keys for arrays, which basically means our current plugin version files will no longer work. See here: https://github.com/symfony/yaml/blame/1ff992f3a74b4c77fea5fd9f382a083915c214aa/Parser.php#L215
They deprecated it in 3.4.0 as shown here: https://github.com/symfony/yaml/commit/240ab5d231719d40a20d8433914b9ea27981624a